src/app/hooks/inbox-linshare-presending-hook.service.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+angular.module('linagora.esn.unifiedinbox.linshare')
+  .factory('inboxLinsharePresendingHook', inboxLinsharePresendingHook);
+
+function inboxLinsharePresendingHook(
+  $q,
+  _,
+  esnI18nService,
+  linshareApiClient,
+  emailSendingService,
```

Comment:
explicit dependencies?

Reply:
There'll be another PR to handle explicit dependencies & translation issues.